Marlin/Marlin_main.cpp

```diff
@@ -11133,6 +11127,10 @@ void setup() {
   #endif
 
   #if HAS_CASE_LIGHT
+    case_light_brightness = 0;   // on power up or reset, default to off
+    #if ENABLED(CASE_LIGHT_DEFAULT_ON)
+      case_light_brightness = 255;
+    #endif
```

Comment:
Generally, to "reduce redundancy" the style has been to break a single code line:
```cpp
case_light_brightness =
  #if ENABLED(CASE_LIGHT_DEFAULT_ON)
    255
  #else
    0
  #endif
;
```
I wish we could use `ENABLED` inside code expressions. It would simplify things.